test/mix_test_watch/port_runner/port_runner_test.exs

```diff
@@ -1,28 +1,58 @@
 defmodule MixTestWatch.PortRunnerTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case, async: false
 
   alias MixTestWatch.PortRunner
   alias MixTestWatch.Config
 
+  setup context do
+    if (Map.has_key?(context, :ansi)) do
+      old_state = Application.get_env(:elixir, :ansi_enabled)
+      Application.put_env(:elixir, :ansi_enabled, context.ansi)
+      on_exit(fn -> Application.put_env(:elixir, :ansi_enabled, old_state) end)
+    end
+
+    context
+  end
+
   describe "build_tasks_cmds/1" do
-    test "appends commandline arguments from passed config" do
```

Comment:
Please don't remove the existing tests, it makes it hard to understand what has changed and whether there are regressions.